dpe/src/commands/derive_context.rs

```diff
@@ -91,6 +93,14 @@ impl DeriveContextCmd {
         self.flags.contains(DeriveContextFlags::RECURSIVE)
     }
 
+    pub const fn allows_exports_cdi(&self) -> bool {
+        self.flags.contains(DeriveContextFlags::EXPORT_CDI)
+    }
+
+    pub const fn allows_create_certificate(&self) -> bool {
+        self.flags.contains(DeriveContextFlags::CREATE_CERTIFICATE)
+    }
```

Comment:
nit: previously the function names used "allows" because the flags would allow specific behavior that you then had to use when calling CertifyKey.

But these flags impact the DeriveContext call directly. So I think we should change these function names to be `creates_certificate` and `exports_cdi`.

Reply:
Updated!